test/data_processing/test_nodes.py

```diff
@@ -240,3 +245,19 @@ def test_iq_averaging(self):
 
         self.assertTrue(np.allclose(avg_datum, expected_avg))
         self.assertTrue(np.allclose(error, expected_std))
+
+
+class TestProbability(QiskitTestCase):
+    """Test probability computation."""
+
+    def test_variance_not_zero(self):
```

Comment:
Can you add a few extra tests to check that we get the right probabilities? E.g. if I give it {"1": 50, "0": 50} with a flat prior I should get p = 50%.